### DISCUSSION TITLE
Is a replacement needed for null-ls?

### DISCUSSION THREAD

**Discussion by sdondley** [LazyVim/LazyVim#1327] [Q&A]
According to the author, null-ls will no longer be maintained: https://github.com/jose-elias-alvarez/null-ls.nvim/issues/1621

I'm guessing a suitable replacement will be needed?

**Comment by sdondley** [comment]
This persons says they have found a replacement: https://www.reddit.com/r/neovim/comments/15oue2o/finally_a_robust_autoformatting_solution/

**Reply by dpetka2001** [reply]
You can take a look in #1080. Some ppl mention they dabbled with one or other. From reddit I see most ppl heading towards either `efm` or `formatter.nvim`. I will wait until the maintainer comes back and read his thoughts as well. I use Lazyvim entirely, so I will stick with the option it will likely provide or something else if such an option will not be made available by the base distribution (at least until something breaks `null-ls` entirely, cuz until now I haven't had any problems).

**Comment by fredrikaverpil** [comment]
You can use none-ls (fork of null-ls). [According to folke](https://github.com/LazyVim/LazyVim/discussions/1522#discussioncomment-7153402), this will be the default, at least for a few weeks...

**Reply by folke** [reply]
none-ls is already the default in LazyVim, so you don't need to do anything special to start using it.
